docs/source/filters.rst

```diff
@@ -115,6 +116,34 @@ JMESPath Filter
        op: lt
        value: 2
 
+      # This policy will use `intersect` op to compare rds instances subnet group list
+      # against a user provided list of public subnets from a s3 txt file.
+      - name: find-rds-on-public-subnets-using-s3-list
+        comment:  |
+           The txt file needs to be in utf-8 no BOM format and contain one
+           subnet per line in the file no quotes around the subnets either.
+        resource: rds
+        filters:
+            - type: value
+              key: "DBSubnetGroup.Subnets[].SubnetIdentifier"
+              op: intersect
+        value_from:
```

Comment:
value_from indentation should line up with the filter

Reply:
Whoops, sorry didn't even notice that formatting got messed up.  Corrected it.  Thanks